schemas/book.schema.tpl.json

```diff
@@ -0,0 +1,17 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/Book",
+  "_extends": "creativeWork.schema.tpl.json",
+  "required": [
+      "fullName"
+  ],
+  "properties": {
+    "digitalIdentifier": {
+      "_instruction": "Add the globally unique and persistent digital identifier of this research product. Note that this digital identifier will be used to reference all attached research product versions.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/ISBN",
+        "https://openminds.ebrains.eu/core/DOI"
+      ],
+      "_originalName": "isbn"
```

Comment:
???

Reply:
sorry, the "_originalName" field should have been deleted. Will be gone in the next push.

These schemas are closely based on schema.org, and where I changed a field name to use an equivalent openMINDS term I used this field to keep a note for myself of the original schema.org name, but the field wasn't intended to go into the v1 branch.